Tella/Data/MainAppModel.swift

```diff
@@ -107,6 +107,14 @@ class MainAppModel: ObservableObject, AppModelFileManagerProtocol {
         vaultManager.removeAllFiles()
         publishUpdates()
     }
+
+    func deleteAllServersConnection() {
```

Comment:
I move this method from serverModel to MainAppModel so we can use it in multiple places without the need of importing server model. Also is using `vaultManager` which corresponds to the appModel so I think it makes sense to be inside the appModel. 

Let me know what do you think @dhekra-rouatbi @RobertShrestha